packages/@lwc/engine-core/src/framework/vnodes.ts

```diff
@@ -45,8 +45,8 @@ export interface VComment extends BaseVNode {
 
 export interface VBaseElement extends BaseVNode {
     sel: string;
-    data: VElementData;
-    children: VNodes;
+    data: Readonly<VElementData>;
```

Comment:
The `Readonly` type utils is defined as following:

```
type Readonly<T> = {
    readonly [P in keyof T]: T[P];
};
```

Since all the properties in `VElementData` are already readonly, there is no need wrap it with `Readonly<>`.

```suggestion
    data: VElementData;
```

Same observation for the usages of `VElementData` in `api.ts`.

Reply:
Done.